lapack-reference:x64-windows build failure

**Host Environment**
- OS: `Windows 10 20H1`;
- Compiler: `MSVC 2019 16.7.1`.

**To Reproduce**
Steps to reproduce the behavior:
```
vcpkg install lapack-reference:x64-windows
```

**Failure logs** 
Console output:
```
Computing installation plan...
The following packages will be built and installed:
    lapack-reference[blas-select,core,noblas]:x64-windows
Detecting compiler hash for triplet x64-windows...
Starting package 1/1: lapack-reference:x64-windows
Building package lapack-reference[blas-select,core,noblas]:x64-windows...
Could not locate cached archive: C:\Users\spindensity\AppData\Local\vcpkg\archives\96\96b8e768289265c0488a2815d3479b66070e2d56.zip
-- Using cached D:/Repository/vcpkg/downloads/Reference-LAPACK-lapack-v3.8.0.tar.gz
-- Cleaning sources at D:/Repository/vcpkg/buildtrees/lapack-reference/src/v3.8.0-954f10683f.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/Repository/vcpkg/downloads/Reference-LAPACK-lapack-v3.8.0.tar.gz
-- Using source at D:/Repository/vcpkg/buildtrees/lapack-reference/src/v3.8.0-954f10683f.clean
-- The Fortran compiler identification is unknown
-- No Fortran compiler found on the PATH. Using MinGW gfortran!
-- Acquiring MSYS Packages from D:/Repository/vcpkg/downloads/tools/msys2-20200812...
-- Acquiring MSYS Packages... OK
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Fixing pkgconfig
-- Acquiring MSYS Packages from D:/Repository/vcpkg/downloads/tools/msys2-20200812...
-- Acquiring MSYS Packages... OK
-- Fixing pkgconfig - release
-- Checking file: D:/Repository/vcpkg/packages/lapack-reference_x64-windows/lib/pkgconfig/lapack.pc
-- pkg_cfg_cmd call with:D:/Repository/vcpkg/downloads/tools/msys2-20200812/msys64/usr/bin/pkg-config.exe --exists lapack failed
-- ENV{PKG_CONFIG_PATH}:/D/Repository/vcpkg/installed/x64-windows/lib/pkgconfig:/D/Repository/vcpkg/installed/x64-windows/share/pkgconfig:/D/Repository/vcpkg/packages/lapack-reference_x64-windows/lib/pkgconfig:/D/Repository/vcpkg/packages/lapack-reference_x64-windows/share/pkgconfig
-- pkg-config call failed with error code:1
-- pkg-config output:
CMake Error at scripts/cmake/vcpkg_fixup_pkgconfig.cmake:84 (message):
  pkg-config error output:Package blas was not found in the pkg-config search
  path.

  Perhaps you should add the directory containing `blas.pc'

  to the PKG_CONFIG_PATH environment variable

  Package 'blas', required by 'lapack', not found
Call Stack (most recent call first):
  scripts/cmake/vcpkg_fixup_pkgconfig.cmake:316 (vcpkg_fixup_pkgconfig_check_files)
  ports/lapack-reference/portfile.cmake:94 (vcpkg_fixup_pkgconfig)
  scripts/ports.cmake:79 (include)


Error: Building package lapack-reference:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: lapack-reference:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.
```

**Additional context**
It's not the same as #12888. The error happened after pull request https://github.com/Jimmy-Hu/vcpkg/pull/285 had been merged.

> in port file or triplet str
> 
> `set(PORT_DEBUG ON)`
> 
> `vcpkg install lapack-reference --triplet x64-windows --debug`
> 
> log file attach

## Console output

```
[DEBUG] Feature flag 'binarycaching' unset
[DEBUG] Feature flag 'manifests' unset
[DEBUG] Feature flag 'compilertracking' unset
[DEBUG] Using vcpkg-root: D:\Repository\vcpkg
[DEBUG] Using installed-root: D:\Repository\vcpkg\installed
[DEBUG] Using buildtrees-root: D:\Repository\vcpkg\buildtrees
[DEBUG] Using downloads-root: D:\Repository\vcpkg\downloads
[DEBUG] Using packages-root: D:\Repository\vcpkg\packages
[DEBUG] Using scripts-root: D:\Repository\vcpkg\scripts
[DEBUG] Default binary cache path is: C:\Users\spindensity\AppData\Local\vcpkg\archives
Computing installation plan...
[DEBUG] CreateProcessW("D:\Repository\vcpkg\downloads\tools\cmake-3.17.2-windows\cmake-3.17.2-win32-x86\bin\cmake.exe" --version)
[DEBUG] cmd_execute_and_stream_data() returned 0 after    33901 us
[DEBUG] CreateProcessW("D:\Repository\vcpkg\downloads\tools\cmake-3.17.2-windows\cmake-3.17.2-win32-x86\bin\cmake.exe" -DVCPKG_ROOT_DIR=D:/Repository/vcpkg -DPACKAGES_DIR=D:/Repository/vcpkg/packages -DBUILDTREES_DIR=D:/Repository/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=D:/Repository/vcpkg/installed -DDOWNLOADS=D:/Repository/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "D:\Repository\vcpkg\buildtrees\0.vcpkg_dep_info.cmake")
[DEBUG] cmd_execute_and_stream_data() returned 0 after    20536 us
[DEBUG] CreateProcessW("D:\Repository\vcpkg\downloads\tools\cmake-3.17.2-windows\cmake-3.17.2-win32-x86\bin\cmake.exe" -DVCPKG_ROOT_DIR=D:/Repository/vcpkg -DPACKAGES_DIR=D:/Repository/vcpkg/packages -DBUILDTREES_DIR=D:/Repository/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=D:/Repository/vcpkg/installed -DDOWNLOADS=D:/Repository/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "D:\Repository\vcpkg\buildtrees\1.vcpkg_dep_info.cmake")
[DEBUG] cmd_execute_and_stream_data() returned 0 after    21187 us
[DEBUG] CreateProcessW("D:\Repository\vcpkg\downloads\tools\cmake-3.17.2-windows\cmake-3.17.2-win32-x86\bin\cmake.exe" -DVCPKG_ROOT_DIR=D:/Repository/vcpkg -DPACKAGES_DIR=D:/Repository/vcpkg/packages -DBUILDTREES_DIR=D:/Repository/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=D:/Repository/vcpkg/installed -DDOWNLOADS=D:/Repository/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "D:\Repository\vcpkg\buildtrees\0.vcpkg_tags.cmake")
[DEBUG] cmd_execute_and_stream_data() returned 0 after    21075 us
The following packages will be built and installed:
    lapack-reference[blas-select,core,noblas]:x64-windows
[DEBUG] CreateProcessW("C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe" -all -prerelease -legacy -products * -format xml)
[DEBUG] cmd_execute_and_stream_data() returned 0 after    25295 us
Detecting compiler hash for triplet x64-windows...
[DEBUG] Found path: C:\Program Files\Git\cmd\git.exe
[DEBUG] CreateProcessW("D:\Repository\vcpkg\downloads\tools\git-2.26.2-1-windows\mingw32\bin\git.exe" --version)
[DEBUG] cmd_execute_and_stream_data() returned 0 after    22934 us
[DEBUG] CreateProcessW(cmd /c ""C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvarsall.bat" -vcvars_ver=14.2 amd64   2>&1 <NUL" & echo cdARN4xjKueKScMy9C6H& set)
[DEBUG] cmd_execute_and_stream_data() returned 0 after   985536 us
[DEBUG] CreateProcessW("D:\Repository\vcpkg\downloads\tools\cmake-3.17.2-windows\cmake-3.17.2-win32-x86\bin\cmake.exe" -DCURRENT_PORT_DIR=D:/Repository/vcpkg/scripts/detect_compiler -DCURRENT_BUILDTREES_DIR=D:/Repository/vcpkg/buildtrees/detect_compiler -DCURRENT_PACKAGES_DIR=D:/Repository/vcpkg/packages/detect_compiler_x64-windows -DCMD=BUILD -DTARGET_TRIPLET=x64-windows "-DTARGET_TRIPLET_FILE=D:\Repository\vcpkg\triplets\x64-windows.cmake" -DVCPKG_PLATFORM_TOOLSET=v142 -DDOWNLOADS=D:/Repository/vcpkg/downloads -DVCPKG_CONCURRENCY=13 -DGIT=D:/Repository/vcpkg/downloads/tools/git-2.26.2-1-windows/mingw32/bin/git.exe -DVCPKG_ROOT_DIR=D:/Repository/vcpkg -DPACKAGES_DIR=D:/Repository/vcpkg/packages -DBUILDTREES_DIR=D:/Repository/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=D:/Repository/vcpkg/installed -DDOWNLOADS=D:/Repository/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "D:\Repository\vcpkg\scripts\ports.cmake")
[DEBUG] -- Configuring x64-windows
[DEBUG] [1/1] cmd /c "cd .. && "D:/Repository/vcpkg/downloads/tools/cmake-3.17.2-windows/cmake-3.17.2-win32-x86/bin/cmake.exe" "D:/Repository/vcpkg/scripts/detect_compiler" "-DCMAKE_MAKE_PROGRAM=D:/Repository/vcpkg/downloads/tools/ninja/ninja-1.10.0/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=D:/Repository/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=D:/Repository/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=D:/Repository/vcpkg" "-D_VCPKG_INSTALLED_DIR=D:/Repository/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=D:/Repository/vcpkg/packages/detect_compiler_x64-windows""
[DEBUG] #COMPILER_HASH#12e212886f6f3928abf1d2d3d5106df42e436722
[DEBUG] -- Configuring done
[DEBUG] -- Generating done
[DEBUG] CMake Warning:
[DEBUG]   Manually-specified variables were not used by the project:
[DEBUG]
[DEBUG]     BUILD_SHARED_LIBS
[DEBUG]     CMAKE_INSTALL_BINDIR
[DEBUG]     CMAKE_INSTALL_LIBDIR
[DEBUG]     VCPKG_TARGET_ARCHITECTURE
[DEBUG]
[DEBUG]
[DEBUG] -- Build files have been written to: D:/Repository/vcpkg/buildtrees/detect_compiler/x64-windows-rel
[DEBUG]
[DEBUG] cmd_execute_and_stream_data() returned 0 after  1449314 us
[DEBUG]
[DEBUG] Detecting compiler hash for triplet x64-windows: 12e212886f6f3928abf1d2d3d5106df42e436722
[DEBUG] <abientries for lapack-reference:x64-windows>
[DEBUG]   CONTROL|e2f1fc4230a0e3decf03779fe16b17ec52579562
[DEBUG]   FindLAPACK.cmake|194fa5a963debfa39bb9f80c236cb3d9ed6de951
[DEBUG]   cmake|3.17.2
[DEBUG]   features|blas-select;core;noblas
[DEBUG]   openblas|7a610d07156a20dc63c0dfda793dc1f108b7a7c4
[DEBUG]   portfile.cmake|1037fb175fdb391300202886e40a5ea1d86d172d
[DEBUG]   post_build_checks|2
[DEBUG]   powershell|ab1effc926b000a6adc12198a1886514ec203621a53b0cd7ec1cd9a8225dccda7e857feaabcfba4004bea73129b986abaad777c4573f44e0af70411226ce08b0
[DEBUG]   triplet|3eab5c64d9cf628fc0aec277fde66dbdc69b8516-83a62cd5c237de1a32ff2f55ead5e6b66c461200-12e212886f6f3928abf1d2d3d5106df42e436722
[DEBUG]   vcpkg-cmake-wrapper.cmake|17bbe129b18940418a80570acbfc1313cebd7561
[DEBUG]   vcpkg-gfortran|1db3cb7b57d3c0074f15b18ac0a77bfa6414329a
[DEBUG]   vcpkg_configure_cmake|1264ee650203750d3c91272e9da8f8d9de0b948d
[DEBUG]   vcpkg_copy_pdbs|4bbc15802daf8f9f8a92794c10b0a89cfca1e742
[DEBUG]   vcpkg_find_fortran|4341ba8f2573ffa919dbdcd87fae911693ad057b
[DEBUG]   vcpkg_fixup_cmake_targets|ef088786b9a6815971b8b6bd3a511183e77a8b93
[DEBUG]   vcpkg_fixup_pkgconfig|7999b090e55aef0866dda59c6b167c52b55467f5
[DEBUG]   vcpkg_from_git|13f63ff436e043b1297055ca7d59776d9e925cd1
[DEBUG]   vcpkg_from_github|4624dffdf2b8d85ffdfde0850369fa96abdf0bb8
[DEBUG]   vcpkg_install_cmake|75d1079d6f563d87d08c8c5f63b359377aa19a6e
[DEBUG] </abientries>
Starting package 1/1: lapack-reference:x64-windows
Building package lapack-reference[blas-select,core,noblas]:x64-windows...
Could not locate cached archive: C:\Users\spindensity\AppData\Local\vcpkg\archives\e7\e772ea9fcf0b4631cb332f5be354b02efb4e0cb9.zip
[DEBUG] CreateProcessW("D:\Repository\vcpkg\downloads\tools\cmake-3.17.2-windows\cmake-3.17.2-win32-x86\bin\cmake.exe" -DPORT=lapack-reference -DCURRENT_PORT_DIR=D:/Repository/vcpkg/ports/lapack-reference -DVCPKG_USE_HEAD_VERSION=0 -D_VCPKG_NO_DOWNLOADS=0 -D_VCPKG_DOWNLOAD_TOOL=BUILT_IN -D_VCPKG_EDITABLE=0 "-DFEATURES=blas-select;core;noblas" "-DALL_FEATURES=blas-select;cblas;noblas;" -DCMD=BUILD -DTARGET_TRIPLET=x64-windows "-DTARGET_TRIPLET_FILE=D:\Repository\vcpkg\triplets\x64-windows.cmake" -DVCPKG_PLATFORM_TOOLSET=v142 -DDOWNLOADS=D:/Repository/vcpkg/downloads -DVCPKG_CONCURRENCY=13 -DGIT=D:/Repository/vcpkg/downloads/tools/git-2.26.2-1-windows/mingw32/bin/git.exe -DVCPKG_ROOT_DIR=D:/Repository/vcpkg -DPACKAGES_DIR=D:/Repository/vcpkg/packages -DBUILDTREES_DIR=D:/Repository/vcpkg/buildtrees -D_VCPKG_INSTALLED_DIR=D:/Repository/vcpkg/installed -DDOWNLOADS=D:/Repository/vcpkg/downloads -DVCPKG_MANIFEST_INSTALL=OFF -P "D:\Repository\vcpkg\scripts\ports.cmake")
-- Using cached D:/Repository/vcpkg/downloads/Reference-LAPACK-lapack-v3.8.0.tar.gz
-- Cleaning sources at D:/Repository/vcpkg/buildtrees/lapack-reference/src/v3.8.0-954f10683f.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source D:/Repository/vcpkg/downloads/Reference-LAPACK-lapack-v3.8.0.tar.gz
-- Using source at D:/Repository/vcpkg/buildtrees/lapack-reference/src/v3.8.0-954f10683f.clean
-- The Fortran compiler identification is unknown
-- No Fortran compiler found on the PATH. Using MinGW gfortran!
-- Acquiring MSYS Packages from D:/Repository/vcpkg/downloads/tools/msys2-20200812...
-- Acquiring MSYS Packages... OK
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
-- Fixing pkgconfig
-- Acquiring MSYS Packages from D:/Repository/vcpkg/downloads/tools/msys2-20200812...
-- Acquiring MSYS Packages... OK
-- [DEBUG] Using pkg-config from: D:/Repository/vcpkg/downloads/tools/msys2-20200812/msys64/usr/bin/pkg-config.exe
-- Fixing pkgconfig - release
-- [DEBUG] Files: D:/Repository/vcpkg/packages/lapack-reference_x64-windows/lib/pkgconfig/lapack.pc
-- Checking file: D:/Repository/vcpkg/packages/lapack-reference_x64-windows/lib/pkgconfig/lapack.pc
-- [DEBUG] Checking package (RELEASE): lapack
-- pkg_cfg_cmd call with:D:/Repository/vcpkg/downloads/tools/msys2-20200812/msys64/usr/bin/pkg-config.exe --exists lapack failed
-- ENV{PKG_CONFIG_PATH}:/D/Repository/vcpkg/installed/x64-windows/lib/pkgconfig:/D/Repository/vcpkg/installed/x64-windows/share/pkgconfig:/D/Repository/vcpkg/packages/lapack-reference_x64-windows/lib/pkgconfig:/D/Repository/vcpkg/packages/lapack-reference_x64-windows/share/pkgconfig
-- pkg-config call failed with error code:1
-- pkg-config output:
CMake Error at scripts/cmake/vcpkg_fixup_pkgconfig.cmake:84 (message):
  pkg-config error output:Package blas was not found in the pkg-config search
  path.

  Perhaps you should add the directory containing `blas.pc'

  to the PKG_CONFIG_PATH environment variable

  Package 'blas', required by 'lapack', not found
Call Stack (most recent call first):
  scripts/cmake/vcpkg_fixup_pkgconfig.cmake:316 (vcpkg_fixup_pkgconfig_check_files)
  ports/lapack-reference/portfile.cmake:96 (vcpkg_fixup_pkgconfig)
  scripts/ports.cmake:79 (include)


[DEBUG] cmd_execute_and_stream_data() returned 1 after 136066565 us
Error: Building package lapack-reference:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: lapack-reference:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.
[DEBUG] D:\Repository\vcpkg\toolsrc\src\vcpkg\install.cpp(492)
[DEBUG] Exiting after 138780679 us (138780559 us)
```

## Log files

[lapack-reference-logfiles.zip](https://github.com/microsoft/vcpkg/files/5086053/lapack-reference-logfiles.zip)

> merge patch #12956
> reinstall openblas

Could you please clarify what do you mean by "merge patch". I am using Windows 10 and running these command on cmd. 

Also this is the detail of the problem   

CMake Error at scripts/cmake/vcpkg_fixup_pkgconfig.cmake:74 (message):
  pkg-config error output:Package blas was not found in the pkg-config search
  path.

  Perhaps you should add the directory containing `blas.pc'

  to the PKG_CONFIG_PATH environment variable

  Package 'blas', required by 'lapack', not found

Error: Building package lapack-reference:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: lapack-reference:x64-windows
  Vcpkg version: 2020.06.15-nohash

--- a/ports/openblas/CONTROL
+++ b/ports/openblas/CONTROL
@@ -1,6 +1,6 @@
 Source: openblas
 Version: 0.3.9
-Port-Version: 2
+Port-Version: 3
 Homepage: https://github.com/xianyi/OpenBLAS
 Build-Depends: pthread (linux&osx)
 Description: OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version.

--- a/ports/openblas/fix-pkg-config.patch
+++ b/ports/openblas/fix-pkg-config.patch
@@ -1,0 +1,19 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index c324e224..4b82d767 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -389,11 +389,9 @@ if(NOT NO_LAPACKE)
+ 	install (FILES ${CMAKE_BINARY_DIR}/lapacke_mangling.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/openblas${SUFFIX64})
+ endif()
+ 
+-include(FindPkgConfig QUIET)
+-if(PKG_CONFIG_FOUND)
+-	configure_file(${PROJECT_SOURCE_DIR}/cmake/openblas.pc.in ${PROJECT_BINARY_DIR}/openblas${SUFFIX64}.pc @ONLY)
+-	install (FILES ${PROJECT_BINARY_DIR}/openblas${SUFFIX64}.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig/)
+-endif()
++# Install pkg-config files
++configure_file(${PROJECT_SOURCE_DIR}/cmake/openblas.pc.in ${PROJECT_BINARY_DIR}/openblas${SUFFIX64}.pc @ONLY)
++install (FILES ${PROJECT_BINARY_DIR}/openblas${SUFFIX64}.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig/)
+ 
+ 
+ # GNUInstallDirs "DATADIR" wrong here; CMake search path wants "share".

--- a/ports/openblas/portfile.cmake
+++ b/ports/openblas/portfile.cmake
@@ -9,6 +9,7 @@ vcpkg_from_github(
         fix-space-path.patch
         fix-redefinition-function.patch
         github_2481.patch
+        fix-pkg-config.patch
 )
 
 find_program(GIT NAMES git git.cmd)

